### ISSUE TITLE
Nurture/Nourish belly mode

### ISSUE THREAD

**Issue by Dinnel** [VOREStation/VOREStation#789]
#### Brief description of the issue
I was wondering if there can be a nurture mode. I may of mentioned it under suggestions but this is the first time I'm asking as an issue itself. If I had to sum it up, its basically the opposite of draining and while not outright similar to healing mode, it can be used for something like it.

#### How do I think Nurture mode should work?
Note that some of this is probably not a good idea, just throwing stuff out there to make the mode slightly flexible.

The first and the biggest feature I'm hoping gets put in is to share nutriment. This can be either adjusted so the 'pred' gives the 'prey' nutrients if they're lower than the pred's nutriment level or alternatively, if either one of them is out of balance, the mode will try to equalize.

The second feature would be sharing chemicals, either splitting it between the pred and prey (So for example you could take 30 units of water and it would be split evenly between the pred and prey) or make sure the prey has roughly the same amount of chems the pred has (So if the pred has 30 units of water, so would all the prey inside the pred)

The third, I would suggest sharing toxin damage, perhaps strictly from prey to pred. It would work like if the prey has 20 tox damage. and the pred has 10 tox damage, the pred would slowly draw tox damage from the prey until he matches, so the pred would have 15 tox damage after the process is done while the prey would also have 15 tox damage. If the pred had 0 to start with, they'd have 10 and the prey would have 10 after it's all done and over with.

Finally, as long as the pred is able to breathe, prey will be immune to oxyloss in the case of lungs being ruptured, bloodloss or people in crit (Though I think thats already in, not sure)

#### Summary
In short, a mode that treats the 'prey' like a child and the 'pred' as the mother, sharing the same thing the pred ingests as well as nutriment. It doesn't heal physical injuries directly like heal mode, but 'heals' toxin by sharing it with the pred and prevents oxyloss. It can also run a bit differently if someone else has a good idea on how it should work.

**Comment by SpadesNeil** [comment]
Kind of a niche idea, but I can see why there'd be an appeal in it. Not sure if it's worth adding but let's talk about it.

**Comment by Dinnel** [comment]
Someone suggested in development to rename it to **nourish** instead of **nurture**. It does sound kind of better.

**Comment by Screemonster** [comment]
This'd be a pretty useful thing to have after absorption so the people don't _have_ to be hungry to the point of starvation when you let them out.

**Comment by Dinnel** [comment]
More discussion about in general chat. Another person thought of a solution to the chemical transfer bit.

What they had in mind was that the transfer should instead of splitting it something like 50/50 over the course of time, it instead should softly trickle down, which would help prevent ODing on chemicals if the pred decides to binge on something like bicard (which would actually screw over the prey if they had internal bleeding, the system however would help them keep in an overdosed state if they were in one before without any more additional injections on the prey's part; though why you're using vore in an emergency is beyond me.)

To explain their vision, it would be to check every chemical in the pred's system, such as water, alcohol, chemicals, you name it; all of it gets checked, then each one has its metabolism rate examined. It then takes this rate and transfers the amount into the prey from the pred. So for example, the pred had 20 units of a chemical and it metabolized by .5 per tick, it would mean another .5 would be taken from them and then be added to the prey inside, in which the tick would quickly metabolize it. This means by the next tick, the pred would have 19 units and the prey would have .5, then its metabolized. Then the next tick comes its 18 units and the prey STILL has .5 .

**Comment by SpadesNeil** [comment]
Did this ever get added or are we just not doing this?

**Comment by Dinnel** [comment]
I could give a try at doing it myself, I almost forgot I even made this suggestion.

**Comment by Screemonster** [comment]
Honestly, I'd probably just make it the opposite of drain-mode and have it pass nutrition from pred to prey until they're equal.

**Comment by Dinnel** [comment]
Alternatively we could fiddle with Heal mode which has something similar to what Scree says but instead of limiting to maximum of 300, the maximum is the current nutrition of the pred.

**Comment by Screemonster** [comment]
oh, if heal mode already nourishes the prey up to 300 then that's fine for most cases (like unabsorbing and wanting to get them off starvation-levels) so I guess a separate mode would only be useful for inflicting THE BLOAT

I guess it's pretty rare for absorbypreds to actually let people out before round-end so it hasn't come up :v

**Comment by Dinnel** [comment]
The bloat IS the fun part! But seriously, I would like to see a mode that transfers reagnants from one person to another for medi-possibilites, promoting a more voracious medical who insist this is better than needles... which might be if the chems copy over to multiple prey or the pred themselves have lower metabolism which means they can extend the chem's duration in faster metabolizing prey or ironically making them fatter than the pred when they eat food.
 Though in reality this may not get used nearly as much due to emergencies and the hassle of looking through prefs to make sure you're eating them right but GOSH DARN IT I'm hoping it gets added in or I find out how to add it in myself!
